### PR TITLE
master layout web

### PR DESCRIPTION
**Pull request by Bayka1997**
==> Title : Master layout web

==> Redmine:
https://edu-redmine.sun-asterisk.vn/issues/16505

==> Attachment File:

### CODE REVIEW

**Comment by Bayka1997** [comment]
ready

**Comment by Bayka1997** [comment]
ready

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<header id="header" id="home">
+  <div class="container">
+    <div class="row align-items-center justify-content-between d-flex">
+      <div id="logo">
+        <%= link_to do %>
+          <%= image_tag "/images/logo.png" %>
```
Comment (phucmh-1356) [inline app/views/layouts/_header.html.erb]:
```suggestion
          <%= image_tag "logo.png" %>
```

**Comment by Bayka1997** [comment]
ready

**Comment by Bayka1997** [comment]
ready

**Pull request merged**